test/jdk/javax/management/remote/mandatory/connection/IdleTimeoutTest.java

```diff
@@ -211,9 +211,8 @@ private static boolean test(String proto,int opCount,int liCount)
                     System.out.println("Operation[" + (i+1)
                                        +"]: starting at " +
                                        elapsed + "ms");
-                    final String name = "d:type=mlet,instance=" + i;
-                    mbs.createMBean("javax.management.loading.MLet",
-                                    new ObjectName(name));
+                    final String name = ":name=Test_instance_" + i;
```

Comment:
Q: The same question, why it is not started with the `d:type=` as it was before?

Reply:
Yes, just an identifier, the content is not very important.
type=mlet should be removed to clarify it's not an m-let, but would not stop it from working.
I could leave this one unchanged other than removing "type=mlet".